tests/src/test/scala/com/nvidia/spark/rapids/GeneratedInternalRowToCudfRowIteratorRetrySuite.scala

```diff
@@ -85,16 +85,13 @@ class GeneratedInternalRowToCudfRowIteratorRetrySuite
       ctriter, schema, TargetSize(Int.MaxValue),
       NoopMetric, NoopMetric, NoopMetric, NoopMetric, NoopMetric))
     RmmSpark.forceRetryOOM(RmmSpark.getCurrentThreadId, 2)
-    assertResult(0)(RmmSpark.getAndResetNumRetryThrow(RmmSpark.getCurrentThreadId))
+    assertResult(0)(RmmSpark.getAndResetNumRetryThrow(1))
```

Comment:
I don't understand why we are using a thread ID of 1? Could you at least add some comments explaining what is happening here?

Reply:
That's right I forgot my own APIs.